packages/bee-q/src/components/notification/bq-notification.tsx

```diff
@@ -0,0 +1,188 @@
+  }
+
+  /**
+   * Trigger function when you want to show Notification
+   */
+  @Method()
+  async show() {
+    this.showNotification();
+  }
+
+  // Local methods
+  // Internal business logic.
+  // These methods cannot be called from the host element.
+  // =======================================================
+
+  private getNotificationIconName = () => {
+    if (!this.showIcon) return '';
+
+    const type = this.type;
+    let icon = 'info';
+
+    if (type === 'success') {
+      icon = 'check-circle';
+    }
+
+    if (type === 'error') {
+      icon = 'x-circle';
+    }
+
+    return icon;
+  };
+
+  private getNotificationColor = () => {
+    const type = this.type;
+    const colors = {
+      default: 'ui--inverse-active',
+      success: 'ui--success',
+      info: 'ui--brand',
+      warning: 'ui--warning',
+      error: 'ui--danger',
+    };
+
+    return isDefined(this.subjectColor) ? this.subjectColor : colors[type];
+  };
+
+  private hideNotification = () => {
+    this.isHidden = true;
+  };
+
+  private showNotification = () => {
+    this.isHidden = false;
+  };
+
+  // render() function
+  // Always the last one in the class.
+  // ===================================
+
+  render() {
+    const styles = {
+      ...(this.subjectColor && { color: getColorCSSVariable(this.subjectColor) }),
+    };
+
+    return (
+      <Host style={styles} aria-hidden={this.isHidden} hidden={this.isHidden}>
+        <div
+          class="bg-white notification-shadow notification-radius inline-block w-auto min-w-[var(--bq-notification--min-width)] p-[var(--bq-notification--padding)]"
+          part="base"
+        >
+          {this.showIcon && (
+            <div class="mr-2 inline-block text-left align-top" part="icon">
+              <bq-icon name={this.getNotificationIconName()} color={this.getNotificationColor()} size="24"></bq-icon>
+            </div>
+          )}
+          {!this.showIcon && (
+            <div class="avatar-slot-holder inline-block text-left align-top" part="avatar">
+              <slot name="avatar" />
+            </div>
+          )}
+          <div class="inline-block max-w-xs text-left align-top">
+            <div class="title-font font-semibold" part="title">
+              <slot />
+            </div>
+            <div class="description-slot-holder description-font font-regular" part="description">
+              <slot name="description" />
+            </div>
+            <div class="footer-container" part="footer">
+              <slot name="notification-footer" />
+            </div>
+          </div>
+          {this.showClose && (
+            <div class="w-notification-close inline-block text-right align-top" part="close-icon">
+              <bq-icon
+                name="x"
+                color="icon--primary"
+                size="14"
+                class="cursor-pointer"
+                onClick={this.hideNotification}
+              ></bq-icon>
+            </div>
+          )}
```

Comment:
I believe that the close button icon should be a bit up in the HTML layout hierarchy:

```jsx
<Host
  class={{ '!hidden': !this.isOpen }}
  aria-hidden={!this.isOpen ? 'true' : 'false'}
  hidden={!this.isOpen ? 'true' : 'false'}
>
  <div
    class="relative inline-flex min-w-[var(--bq-notification--min-width)] items-start rounded-[var(--bq-notification--border-radius)] bg-bg-primary p-[var(--bq-notification--padding)] shadow-m"
    part="base"
  >
    {/* CLOSE BUTTON */}
    {!this.disableClose && (
      <bq-button
        class="notification--close absolute right-5"
        appearance="text"
        size="small"
        onClick={() => this.hide()}
        part="btn-close"
      >
        <bq-icon name="x" />
      </bq-button>
    )}
    {/* ICON */}
    {!this.hideIcon && (
      <div
        class={{
          'notification--icon mr-2 inline-block text-left align-top': true,
          [`color-${this.type}`]: true, // The icon color will be based on the type (info, success, warning, error)
        }}
      >
        {!this.hasCustomIcon ? <bq-icon name={this.iconName} /> : <slot name="icon" />}
      </div>
    )}
    {/* CONTENT */}
    <div class="flex flex-col items-start gap-4" part="content">
      {/* TITLE */}
      <div class="title-font font-semibold leading-large" part="title">
        <slot />
      </div>
      {/* BODY */}
      <div class="text-s leading-regular" part="body">
        <slot name="body" />
      </div>
      {/* FOOTER */}
      <div class="flex items-start gap-2" part="footer">
        <slot name="footer" />
      </div>
    </div>
  </div>
</Host>
```

That will guarantee that moving focus around with the Tab key will get the right order.